src/Data/Spectres.lua

```diff
@@ -2422,6 +2632,26 @@ minions["Metadata/Monsters/Spiders/SpiderThornViperStrikeFlickerStrike"] = {
 	modList = {
 	},
 }
+minions["Metadata/Monsters/Spiders/SpiderThornViperStrikeFlickerStrikeWeaver"] = {
+	name = "Deadly Tarantula",
```

Comment:
This ends up having the same name as the spider above it.  Do we know which is which and can we override one or both of the names to clarify the difference?